mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxManagerTest.java

```diff
@@ -20,8 +20,8 @@
 
 import org.apache.james.mailbox.store.mail.model.AbstractMailboxMapperTest;
 
-public class CassandraMailboxMapperTest extends AbstractMailboxMapperTest {
```

Comment:
Why this rename ? Here the mailboxMapper is tested.... Your new name is misleading.